### PR TITLE
Add swagger json name option

### PR DESCRIPTION
**Pull request by briantenazas**
- Adds a flag to protoc-gen-swagger to have the generated swagger file use JSON names (camelCased) instead of the raw proto field names, which are usually snake_cased.

### CODE REVIEW

**Comment by briantenazas** [comment]
@aldelucca1 can you take a look when you get a chance please?

**Comment by aldelucca1** [comment]
@briantenazas doh sorry this took so long, if you could just bump the build number in `vars.sh` we can get this merged

**Comment by briantenazas** [comment]
@aldelucca1  done!

File: all/entrypoint.sh [hunk 1]
```diff
@@ -172,6 +174,10 @@ while test $# -gt 0; do
             CSHARP_OPT=$1
             shift
             ;;
+        ----with-swagger-json-names)
```
Comment (aldelucca1) [inline all/entrypoint.sh:177]: Think there are actually too many `-` here, should probably be
```suggestion
        --with-swagger-json-names)
```

Comment (briantenazas) [reply all/entrypoint.sh]: thanks!

Comment (briantenazas) [reply all/entrypoint.sh]: @aldelucca1  should be all set. passes build check

**Pull request merged**